hack/run-local-with-port-forward.sh

```diff
@@ -23,10 +23,15 @@ echo Starting hive port-forward
 kubectl -n "$METERING_NAMESPACE" \
     port-forward "svc/hive-server" 9992:10000 &
 
-echo Starting Prometheus port-forward
-kubectl -n "$METERING_PROMETHEUS_NAMESPACE" \
-    port-forward "svc/${METERING_PROMETHEUS_SVC}" \
-    9993:"${METERING_PROMETHEUS_SVC_PORT}" &
+if [[ -v METERING_PROMETHEUS_HOST ]]; then
```

Comment:
Let's just make it explicit to checking for portforwarding and just have it `if [ "$METERING_PORT_FORWARD_PROMETHEUS" == "true" ]; then` and we can remove the else case. Then set `METERING_PORT_FORWARD_PROMETHEUS` to true by default around line 13/14